uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Steps/1024.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+trait CRM_HRCore_Upgrader_Steps_1024 {
+
+  /**
+   * Remove Social Account Options and reorder de remaining
+   */
+  public function upgrade_1024() {
+    $this->up1024_removeSocialAccountOptions([
+      'Work',
+      'MySpace',
+      'Vine',
+      'Google+',
+      'Snapchat',
+      'Tumblr',
+    ]);
+    $this->up1024_reorderSocialAccountOptions([
+      'LinkedIn',
+      'Twitter',
+      'Facebook',
+    ]);
+
+    return TRUE;
+  }
+
+  /**
+   * Removes Some Social Account Options
```

Comment:
Seems you might need to update this as the requirement is to `delete if not used` or `disable if used`